.vscode-test.js

```diff
@@ -50,17 +49,17 @@ module.exports = defineConfig({
             mocha: {
                 ui: "tdd",
                 color: true,
-                // so doesn't timeout when breakpoint is hit
-                timeout: isDebugRun ? Number.MAX_SAFE_INTEGER : 3000,
+                timeout,
                 forbidOnly: isCIBuild,
+                slow: 100
             },
             reuseMachineInstall: !isCIBuild,
         },
         // you can specify additional test configurations, too
     ],
     coverage: {
         includeAll: true,
-        include: ["**/src/**"],
-        reporter: ["text", "html"],
+        exclude: ["**/test/**"],
```

Comment:
This now pulls in all the node_modules files, intentional?

Reply:
maybe need to clear old "out" and "dist" directories? The vscode-test-cli tool automates adding node_modules

Reply:
Yeah I think you mentioned this before, seems to be reported already:
https://github.com/microsoft/vscode-test-cli/issues/40
https://github.com/microsoft/vscode-test-cli/issues/37

Reply:
verified not in coverage report